modules/ModuleBuilder/MB/MBPackage.php

```diff
@@ -561,18 +561,18 @@ private function getCustomFieldsManifestForModule($module, &$installdefs)
     private function getCustomMetadataManifestForModule($module, &$installdefs)
     {
         $meta_path = 'custom/modules/' . $module . '/metadata';
+        $working_array = [
+            'editviewdefs.php',
+            'detailviewdefs.php',
+            'quickcreatedefs.php'
+        ];
```

Comment:
Hi @ojs87, @jack7anderson7 why are we only including these files?

Reply:
@jack7anderson7 pointed out what has been stated on the PR description.

> All metadata is copied to this directory, except for listviewdefs.php, regardless if it is in the custom/working directory of the exported module.

> This causes an issue where Studio loads in this working metadata file for dashletviewdefs and never overwrites or deletes it because the working directory is only used for detail view, edit view and quickcreate view.